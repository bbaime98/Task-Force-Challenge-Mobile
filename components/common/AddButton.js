import React from 'react'
import { StyleSheet, TouchableOpacity } from 'react-native'
import { Feather } from '@expo/vector-icons';
import colors from '../../utils/colors';

const AddButton = ({onPress}) => {
    return (
        <TouchableOpacity onPress={onPress} style={styles.container}>
            <Feather name="plus" size={24} color="white" />
        </TouchableOpacity>
    )
}

const styles = StyleSheet.create({
    container: {
        width: 60,
        height: 60,
        borderRadius: 30,
        backgroundColor: colors.dark,
        justifyContent: "center",
        alignItems: "center"
    }
})

export default AddButton
