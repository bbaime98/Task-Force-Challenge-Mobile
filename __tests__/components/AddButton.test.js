import React from 'react';
import renderer from 'react-test-renderer';
import AddButton from '../../components/common/AddButton';

jest.useFakeTimers()

it('should render AddButton component', () => {
    const rendered = renderer.create(<AddButton />).toJSON();
    expect(rendered).toBeTruthy();
});
it('AddButton snapshot', () => {
    const tree = renderer.create(<AddButton />).toJSON();
    expect(tree).toMatchSnapshot();
});
