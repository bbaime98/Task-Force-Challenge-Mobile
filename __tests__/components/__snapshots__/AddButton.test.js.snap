// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0C0D0D",
      "borderRadius": 30,
      "height": 60,
      "justifyContent": "center",
      "opacity": 1,
      "width": 60,
    }
  }
>
  <Text />
</View>
`;
